### DISCUSSION TITLE
Disabling text cmp + help for autocompletion

### DISCUSSION THREAD

**Discussion by loku74** [NvChad/NvChad#2089] [Q&A] [answered]
Hello,

I've been learning to use Neovim for a week now and discover this gem. I've disabled friendly-snippets and made my own. However, I don't like the autocompletion for text (see image)

<img width="193" alt="Capture d’écran 2023-06-06 à 20 31 32" src="https://github.com/NvChad/NvChad/assets/92410060/741145e7-775b-4b92-a136-9d8ff5a66df6">

I'm force to press enter to confirm and then I can press tab to indent my code.
Is there any way to disable it ? thanks :)

2nd question:

I can't figure out how to have basic autocompletion (see images where there should be a lot of basic .h file system) or advanced one where the engine finds the function in includes files and autocomplete automatically with the snippets for all needed variables (I saw that in another nvim config)

<img width="238" alt="Capture d’écran 2023-06-06 à 20 35 45" src="https://github.com/NvChad/NvChad/assets/92410060/436b3d0a-7f16-4fc8-b378-b9e8ea37cafd">

**Comment by siduck** [comment, marked as answer]
@loku74 enable lsp in your custom lspconfig
